huggingface/pytorch/inference/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -149,7 +149,8 @@ RUN pip uninstall -y torch torchvision torchaudio model-archiver multi-model-ser
 RUN pip install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cu117 -U \
```

Comment:
Is this part correct - The link seems to point to cu117 while the dockerfile path and base image say this should be cu118.

Reply:
+1

Reply:
Thanks for the heads up! This seems to be left by reusing the pt 1.13 gpt DLC script, in which we pip installed `torchaudio` and `torchvision` from stable pytorch releases, which is not the case anymore in this DLC.

Indeed it's not having any effect, and we should remove it.